lib/ASTGen/Sources/ASTGen/Macros.swift

```diff
@@ -635,10 +635,11 @@ func expandFreestandingMacroInProcess(
 }
 
 /// Retrieve a syntax node in the given source file, with the given type.
-private func findSyntaxNodeInSourceFile<Node: SyntaxProtocol>(
+func findSyntaxNodeInSourceFile<Node: SyntaxProtocol>(
   sourceFilePtr: UnsafeRawPointer,
   sourceLocationPtr: UnsafePointer<UInt8>?,
-  type: Node.Type
+  type: Node.Type,
+  wantOutermost: Bool = false
```

Comment:
Could you add a comment what, `wantOutermost` does? Also, I think it would also be good to document the other parameters.

Reply:
Yes, I will. I should really move this API somewhere else (e.g., on `ExportedSourceFile`) because it's in an odd place now that I made it non-`private`.